internal/antewrapper/tx_gas_limit_decorator.go

```diff
@@ -17,13 +18,39 @@ func NewTxGasLimitDecorator() TxGasLimitDecorator {
 	return TxGasLimitDecorator{}
 }
 
+// govMsgURLs the MsgURLs of all the governance module's messages.
+// Use getGovMsgURLs() instead of using this variable directly.
+var govMsgURLs []string
+
+// getGovMsgURLs returns govVoteMsgUrls, but first sets it if it hasn't yet been set.
+func getGovMsgURLs() []string {
+	if len(govMsgURLs) != 8 {
```

Comment:
This is really minor, but would it be safer to reverse your condition to govMsgURLs == nil? That makes it a little bit easier to read, and in case the contents of govMsgURLs ever change then this doesn't have to also be updated.

Reply:
Yeah... I waffled on that and even messed it up the first time because I had `4` there instead of `8`.

Both in here and in the x/reward action_builder, I changed the test to ` == nil` and put a comment over it about it.